### ISSUE TITLE
High priority visual accessibility issues

### ISSUE THREAD

**Issue by peterychang** [microsoft/autogen#5633]
Issues:
- [x] (6)	Luminosity contrast ratio of focus indicator on 'Venv' control is less than 3:1.
- [x] (14)	Ensures the contrast between foreground and background colors meets WCAG 2 AA minimum contrast ratio thresholds
- [x] (20)	"Only color is used to indicate the focus indicator on the 'Venv', and 'conda' Tab controls."
- [x] (33)	In 400% zoom mode, horizontal bar is appearing.
- [x] (46)	In high contrast mode, keyboard focus indicator is not visible on Left &right navigation pane, Home icon, Core and tab controls 'Venv' and 'Condo'.
- [x] (54)	Luminosity ratio for the Back to top control is less than the required ratio 4.5:1.
- [x] (55)	In high contrast mode, keyboard focus indicator is not visible on the controls appearing in On this page section.
- [x] (56)	Users are not able to differentiate between selected and not selected state of links present in On this page section in high contrast mode.
